linux/LinuxProcessList.c

```diff
@@ -241,7 +241,7 @@ ProcessList* ProcessList_new(UsersTable* usersTable, Hashtable* pidMatchList, ui
       }
       fclose(statfile);
 
-      if (!btime)
+      if (btime == -1)
```

Comment:
Why not use some local variable to record ´btime´ presence and check on that variable instead. Using a magic value here is just prone to break another time when some unlucky guy has a kernel reporting -1 as boot tiime when cosmic rays just hit the wrong way …

Reply:
We can introduce that when solar activity raises significantly :-)

Reply:
It's gonna take more than just gamma rays ...   :)
```
$ grep btime linux/fs/proc/stat.c 
		"btime %llu\n"
```